src/StatPlots.jl

```diff
@@ -9,11 +9,15 @@ using Distributions
 using DataFrames
 
 import KernelDensity
+import Loess
 @recipe f(k::KernelDensity.UnivariateKDE) = k.x, k.density
 @recipe f(k::KernelDensity.BivariateKDE) = k.x, k.y, k.density
 
 @shorthands cdensity
 
+export groupapply
+export get_summary
```

Comment:
Actually, on a second thought I'm in favour of keeping it also due to performance reasons: for  large datasets it may be inconvenient to have to group it every time. In the implementation of `compute_error = :bootstrap` I see a seizable performance improvement by working with GroupedDataFrames directly rather than putting everything in a big DataFrame and then applying groupby.